new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -58,19 +59,26 @@ public void testInstallingOperator() {
     // imagine that installOperator() will try to install the operator, by creating
     // the kubernetes deployment.  this will complete quickly, and will either be
     // successful or not.
-
     String opns = "opns1";
-    OperatorParams opParams
-        = new OperatorParams().image("weblogic-kubernetes-operator:test_itsimpleoperator")
-          .domainNamespaces(Arrays.asList("domainns1", "domainns2"))
-          .serviceAccount("sa-opns1");
-    boolean success = installOperator("weblogic-operator", opns, opParams);
+    OperatorParams opParams =
+        new OperatorParams().releaseName("weblogic-operator")
+            .namespace(opns)
+            .image("weblogic-kubernetes-operator:test_itsimpleoperator")
+            .domainNamespaces("domainns1, domainns2")
```

Comment:
i'm not sure about this being a string like this - i liked it better the way you had it before, where is was clear it was a list of things, this is just a string that has some kind of special meaning that i don't understand

Reply:
changed it back